pkg/distributor/writefailures/manager.go

```diff
@@ -39,7 +39,8 @@ func (m *Manager) Log(tenantID string, err error) {
 		return
 	}
 
-	if !m.tenantCfgs.LimitedLogPushErrors(tenantID) {
+	if !m.tenantCfgs.LimitedLogPushErrors(tenantID) &&
+		!m.tenantCfgs.LogDuplicateStreamInfo(tenantID) {
```

Comment:
this conditional is redundant (since you're checking this on stream.go already) but moreover, this would mean we won't log any new log lines through the manager if `logDuplicateStreamInfo` is false.

Suggestion: don't add this check here. The manager should just emit the log and ratelimit it. Control over what errors are logged or not should probably be done at an upper level.